src/api/ledger/balance-sheet.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+const convertErrors = utils.common.convertErrors;
+
+function formatBalances({balances, obligations, assets}) {
```

Comment:
I think so. This API generally doesn't copy data from the request to the response.